### ISSUE TITLE
Valkyrien Skies & "Does Potato Tick?" (1.20.1)

### ISSUE THREAD

**Issue by s1ard** [ValkyrienSkies/Eureka#340]
### Mod Name

Does Potato Tick

### This issue occurs when only Valkyrien Skies, addons, and the mod I have specified are installed and no other mods

- [X] I have tested this issue and it occurs with only Valkyrien Skies, addons, and the mod I have specified

### Minecraft Version

1.20.x

### Mod Loader

Forge

### Issue description

The mod breaks when the mod "Does Potato Tick?" (A performance mod regarding entities and ticks) is running, ships can't be piloted properly. 

### Issue reproduction

1. Install Forge 47.2.23
2. Install Valkyrien Skies
3. Install Does Potato Tick?
4. Build / find a boat
5. Place a helm
6. Assemble the ship
7. Right click to pilot it

**Comment by s1ard** [comment]
(Adjusted the issue version to 1.20.x)

**Comment by JustTroniusPlay** [comment]
Is this possible to fix this problem by using DPT? config to add VS entities to mod whitelist?

**Comment by Hex4dec** [comment]
Pretty sure this is an issue on VS2's side. VS2 manages the shipyard (where ships are stored, really really far from the player but they're still in the world)

**Comment by walksanatora** [comment]
yeah vs2 loves to break things on ships that have a distance check
